packages/material-ui/src/Popper/Popper.d.ts

```diff
@@ -1,31 +1,19 @@
 import * as React from 'react';
-import PopperJs, { ReferenceObject } from 'popper.js';
+import { Instance, VirtualElement, ComputedPlacement, Options } from '@popperjs/core';
 import { Omit } from '..';
 import { PortalProps } from '../Portal';
 
-export type PopperPlacementType =
-  | 'bottom-end'
-  | 'bottom-start'
-  | 'bottom'
-  | 'left-end'
-  | 'left-start'
-  | 'left'
-  | 'right-end'
-  | 'right-start'
-  | 'right'
-  | 'top-end'
-  | 'top-start'
-  | 'top';
+export type PopperPlacementType = ComputedPlacement;
```

Comment:
I thought it would be nicer to depend on their types. It would be more correct to use `Placement` instead of `ComputedPlacement` but I wasn't sure if auto placements were left out on purpose.

Reply:
Changed this recently. I'd rather use the correct types